### ISSUE TITLE
Elements in Settings rightrail are not accessible

### ISSUE THREAD

**Issue by gsin11** [KhalisFoundation/sttm-web#1407]
**Describe the bug**
Right rail elements (after pressing settings CTA) are not accessible

**To Reproduce**
1. Go to https://www.sikhitothemax.org/hukamnama
2. Click on **Settings**
3. Check if you can access all the elements of the right rail with your keyboard only.

**Expected behavior**
Use the keyboard:
- After clicking on settings, elements inside the right rail should be clickable and accessible through the keyboard.
- The right rail shouldn't be accessible without being visible.

**Screenshots**
![image](https://user-images.githubusercontent.com/3865313/116211693-5f18af00-a744-11eb-97e5-21692c9b9fdf.png)

**Desktop (please complete the following information):**
All the browsers

**Smartphone (please complete the following information):**
All Mobile and tablet devices

**Additional context**
Add some padding to **Reset to Default** CTA as well.
